Readme.md

```diff
@@ -1,2 +1,76 @@
-Experiments to run liquid services in cloud environments. Try "nomad" first;
-the "docker-compose" method will probably be abandoned.
+# Liquid in a Nomad cluster
+Create a configuration file, `nomad-agent.hcl`, with the following content,
+adapted to your machine in case `eth0` is not the main network interface:
+
+```hcl
+advertise {
+  http = "{{ GetInterfaceIP `eth0` }}"
+  serf = "{{ GetInterfaceIP `eth0` }}"
+}
+
+client {
+  enabled = true
+  network_interface = "eth0"
+}
+```
+
+Increase `vm.max_map_count` to at least `262144`, to make elasticsearch happy -
+see [the official documentation][] for details.
+
+[the official documentation]: https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-prod-mode
+
+Make sure you have Python >= 3.7 installed.
```

Comment:
If we're not using anything from it right now, we will end up using something from it really quick.
This code is under active development, there is no reason for us to try and support older releases.

Reply:
AFAIK our most advanced feature is `f''` strings, but we'd have to check what we use from the standard library. Sometimes it's useful to support old Python versions, not all of us have arch or homebrew, but I'd prefer @gabriel-v's approach, so we always get to play with the latest goodies. @salevajo, since you're responsible for the biggest deployment we have, do you feel that requiring the latest version of Python is a burden?

Reply:
For us devs it's easy to use the latest versions, but for sys-admins it may be a burden, depending on the situation.

Reply:
Ok, the py37 is not being added in this PR, it was already there. The readme file is being moved from `/nomad/Readme.md` to `/Readme.md`. If we want to relax the requirement, we can do it in a separate issue.

Reply:
> @salevajo, since you're responsible for the biggest deployment we have, do you feel that requiring the latest version of Python is a burden?

We should provide support for the Python version included in the currently supported/mentioned Ubuntu release in use. That means for now - using a 16.04.x LTS server installation in production - Python 3.5.x.

Reply:
> We should provide support for the Python version included in the currently supported/mentioned Ubuntu release in use. That means for now - using a 16.04.x LTS server installation in production - Python 3.5.x.

Then please make an issue.